book/website/reproducible-research/reviewing/reviewing-checklist.md

```diff
@@ -1,40 +1,180 @@
+(rr-checklist-for-code-review)=
 ## Checklist for code review process
 
-The following presents some possible checklists for both the coder and the reviewer, as part of a formal review process.
+This section presents some checklists for both the coder and the reviewer, as part of a formal review process.
+The reviewer checklists are split into two categories: one for the whole program, and one for individual files or proposed changes.
+
+The lists are created with a focus on good software engineering practice and are intended to be a source of inspiration.
+When assessing the checklists, it is recommended to consider to what extent the item mentioned is implemented.
+Some items on the lists may not apply to your project or programming language, in which case they should be disregarded.
+
+In all cases, the goal is to use your programming experience to figure out how to make the code better.
 
 ## Table of contents
 
 - [For the coder](#For_the_coder)
 - [For the reviewer](#For_the_reviewer)
+- [Program level checklist](#Program_level_checklist)
+- [File/Change level checklist](#File/Change_level_checklist)
 
 <a name="For_the_coder"></a>
 ### For the coder
 
-- [ ] Does the new code meet project standards? In particular,
-  - [ ] Is there documentation?
-  - [ ] Are there new tests for the new material?
+- [ ] Does the new code meets the required standards of the project?
+  The standards are typically written under `contributing guidelines` by the project you are contributing to.
+- [ ] Is there [documentation](#documentation) to the required standards of the project?
+- [ ] Are you following any declared {ref}`style guide<rr-code-quality>` for the project?
+- [ ] Are there new [tests](#tests) for the new material, based on the required standards of the project?
   - [ ] Do these tests pass locally?
-  - [ ] Are you following any declared style guides?
   - [ ] Are the tests in the rest of the code base still passing locally?
-- [ ] Create the pull request; wait for any CI checks to complete.
-- [ ] Consult the CI reports. Did all the builds and tests complete?
+- [ ] Create the pull request; wait for any {ref}`continuous integration (CI)<rr-ci>` checks to complete.
+- [ ] Consult the CI reports. Did all builds and tests complete? your code is not causing the tests in the main project to fail. Many CI systems will test this automatically for you once you create the PR.
```

Comment:
CI reports can come both from the repository system in used (for instance, GitHub), or custom ones, set up at the beginning by the repository managers. But it can be overkill for beginners managing a software project

Reply:
The item in the checklist is now revised. Does it address your concern?